openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -193,6 +193,40 @@
             charts.push(options);
         }
         charts = sortByTraceOrder(data.trace_order, charts, '_key');
+        if(data.trace_order !== undefined)  {
+            var total_traffic_charts = charts[0].y;
```

Comment:
I think we can stop using `var` and use `let` and `const` only.

`const` support: https://caniuse.com/?search=const
`let` support: https://caniuse.com/?search=let

CC: @nemesisdesign

Reply:
Actually! I did this in my first commit, but to maintain consistency I again updated this.